test/extended/cluster/cl.go

```diff
@@ -74,11 +74,40 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 				e2e.Logf("Our tuning set is: %v", tuning)
 			}
 			for j := 0; j < p.Number; j++ {
-				// Create namespaces as defined in Cluster Loader config
-				nsName := fmt.Sprintf("%s%d", p.Basename, j)
-				err := oc.Run("new-project").Args(nsName).Execute()
+				nsName := p.Basename
+				if len(p.IfExists) == 0 {
```

Comment:
Use a nil check like the rest of the parameters

Reply:
Done